### DISCUSSION TITLE
Time Formatting

### DISCUSSION THREAD

**Discussion by siujd2001** [MMRIZE/MMM-CalendarExt3#179] [General]
I'm having trouble setting the time for the Google calendar.  I have set the locale: en-US and timeformat:24, but the calendar still shows am/pm and not HH:mm.  Raspberry Pi is set for HH:mm time.  Any help would be greatly appreciated.

**Comment by eouia** [comment]
It depends on the `locale` and `eventTimeOptions` in the config.
https://github.com/MMRIZE/MMM-CalendarExt3?tab=readme-ov-file#config-details

<img width="859" alt="image" src="https://github.com/user-attachments/assets/90994fc9-9af8-4f81-841d-fcea2d37ce28">

<img width="859" alt="image" src="https://github.com/user-attachments/assets/6048e9a2-3d33-43f0-a45e-87d6394b7a04">
